pkg/storage/storage.go

```diff
@@ -59,6 +59,13 @@ type PaginationOptions struct {
 	From     string
 }
 
+func (p PaginationOptions) Apply(opts *Options) {
+	if p.PageSize == 0 {
+		p.PageSize = DefaultPageSize
+	}
+	opts.Pagination = &p
+}
+
 // NewPaginationOptions creates a new [PaginationOptions] instance
```

Comment:
did you drop all usages of this function? If so, can you remove it? 

Also, is the godoc on line 56 still accurate?